internal/cli/commands_test.go

```diff
@@ -78,3 +78,41 @@ func TestNextSeq(t *testing.T) {
 		})
 	}
 }
+
+func TestNumDownFromArgs(t *testing.T) {
+	cases := []struct {
+		name                string
+		args                []string
+		applyAll            bool
+		expectedNeedConfirm bool
+		expectedNum         int
+		expectedErrStr      string
+	}{
+		{"no args", []string{}, false, true, -1, ""},
+		{"down all", []string{}, true, false, -1, ""},
+		{"down 5", []string{"5"}, false, false, 5, ""},
+		{"down N", []string{"N"}, false, false, 0, "can't read limit argument N"},
+		{"extra arg after -all", []string{"-all", "5"}, true, false, 0, "-all cannot be used with other arguments"},
```

Comment:
Actually, I think the `args` should be `[]string{"5"}` in this case since `-all` would be captured as a flag.

Did you want to fix this in this PR or in a subsequent one?

Reply:
good call, updated to  `[]string{"5"}`